tests/testthat/test-child-model.R

```diff
@@ -195,7 +195,6 @@ test_that("HIV related deaths among CLHIV not on ART align", {
   dt <- dt %>%
     dplyr::mutate(diff = spec - fr)
 
-  z <- data.table::data.table(dt)
```

Comment:
Does this remove the only usage of data.table? If so, could you take it out of the DESCRIPTION file too.